src/include/sof/math/numbers.h

```diff
@@ -29,7 +29,13 @@
 	__a < 0 ? -1 :		\
 	__a > 0 ? 1 : 0;	\
 })
-
+#ifndef ROUND_DOWN
+#define ROUND_DOWN(size, alignment) ({			\
```

Comment:
@singalsu whats the guidance here now for math macro naming to avoid collisions with Zephyr or C libraries ?